load-test/walkthrough-documents/Create-Full-Production-Size-Static-Test-Data-for-Load-Testing-v02-draft.md

```diff
@@ -0,0 +1,105 @@
+# Create Full Production Size Static Test Data for Load Testiing
+---
+
+## Task Overview
+
+- Create Pupils for each School
+
+
+## Background
+
+
+
+## Create Pupils for each School
+
+
+
+```sql
+
+
+--==============================================================
+-- CREATE PUPILS FOR EACH SCHOOL -- START SQL CODE BLOCK
+
+-- UNIQUE CODE BLOCK IDENTIFIER - STA-MTC-LOAD-TEST-CODE-002v001
+
+-- CODE VERSION v001
+
+
+-- Load Test Data Naming Conventions required for data loading
+
+-- Unique Load Test Data Set Identifier - Format xxx - 'new'
+
+-- 'Teacherxxx1' 'Schoolxxx2' 'Pupilxxx1' - 'Pupilxxx40'
+
+
+-- Set Data Create Variables
+
+DECLARE @testDataSetIdentifier VARCHAR(3) = 'new';
+
+DECLARE @pupilPerSchool INT = 40;
+
+DECLARE @schoolIdStartRange INT = 2;
+
+DECLARE @schoolIdStartRange INT = 3;
```

Comment:
Should be - DECLARE @schoolIDEndRange INT = 3;

if the code is going to be tested first with one loop